javascript/javascript1/week3/traveltime.js

```diff
@@ -0,0 +1,11 @@
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function calculateTime(travelInformation) {
+  const time = travelInformation.destinationDistance / travelInformation.speed;
+  return time;
+}
```

Comment:
You should find `hours` and `minutes` not only the total time. This is a way to do that
```js
  function getTime(speedAndDistance) {
    const totalTime = speedAndDistance.destinationDistance / speedAndDistance.speed;
    const hours = Math.floor(totalTime);
    const minutes = Math.round((totalTime - hours) * 60);
    return `${hours} hours ${minutes} minutes`;
  }
```
You can do the same or find your way